app/scenes/account_notifications/account_notifications.js

```diff
@@ -177,7 +170,6 @@ export default class AccountNotifications extends PureComponent {
     }
 
     saveUserNotifyProps = () => {
-        EventEmitter.emit(SAVING_NOTIFY_PROPS, true);
```

Comment:
I know this will be shown for less than a second but I would rather leave it just to be consistent. make sense?